README.md

```diff
@@ -72,21 +42,37 @@
   </a>
 </p>
 
-<h2>Introduction</h2>
+<br>
+
+## Status
```

Comment:
I would put the badges under then vuetify logo at the top. It is the de-facto standard way of displaying them in readmes.

Reply:
That's how it is currently, I had them nicely laid out.